hack/update-codegen.sh

```diff
@@ -25,6 +25,7 @@ function make_aliases() {
   ( cd mechanics/tools/gen-aliases; go build . )
   # Auto-generate OWNERS_ALIASES from peribolos configs
   ./mechanics/tools/gen-aliases/gen-aliases
```

Comment:
would it make sense to add explicite arguments here, too (like in the line below), so that it's more clear which files are worked on ? (I suspect it's "knativeperibolos/knative.yaml knative-OWNERS_ALIASES" ?)

Reply:
Done.